docs/troubleshooting.md

```diff
@@ -0,0 +1,17 @@
+| Mount successful but files not visible | Try mounting the gcsfuse with --implicit-dir flag. Read the semantics to know the reasoning.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| Mount failed with fusermount3 exit status 1 | It comes when the bucket is already mounted in a folder and we try to mount it again. You need to unmount first and then remount.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Mount failed with error: Current requires cgo or $USER set in environment | It comes when we try mounting by building the gcsfuse codebase. To fix this, build the gcsfuse package by enabling the CGO_ENABLED flag in the go env and then mount back.<ol type="a"><li> Check the current value using - go env command. </li><li>If it is unset, set this using - “export CGO_ENABLED=1” command. </li></ol>                                                                                                                                                                                                                                                                                  |
+| Mount get stuck with error: DefaultTokenSource: google: could not find default credentials | Run ```gcloud auth application-default login``` command to fetch default credentials to the VM. This will fetch the credentials to the following locations: <ol type="a"><li>For linux - $HOME/.config/gcloud/application_default_credentials.json</li><li> %APPDATA%/gcloud/applicateion_default_credentials.json </li></ol>                                                                                                                                                                                                                                                                                       |
```

Comment:
Add "For windows" for options (B) (like "For linux"

Reply:
fixed